code/modules/mob/living/simple_animal/hostile/pitbull.dm

```diff
@@ -91,3 +91,41 @@
 				qdel(src)
 		else
 			gib()
+
+/mob/living/simple_animal/hostile/pitbull/smashednslammed
+	name = "toadline pitbull"
+	icon_state = "toadlinepitbull"
+	icon_living = "toadlinepitbull"
+	icon_dead = "toadlinepitbull_dead"
+	pass_flags = PASSMOB|PASSDOOR //so smashed and slammed they can squeeze through doors
+	health = 35
+	maxHealth = 35
+	speed = 3 //stumpier legs move slower
+
+	melee_damage_lower = 4
+	melee_damage_upper = 6 //stumpy but they hit harder
+
+/mob/living/simple_animal/hostile/pitbull/smashednslammed/New()
+	..()
+	desc = pick(
+		"SMASHED and SLAMMED",
+		"Certified Toadline.",
+		"Help I can't breathe",
+		"Bloodskull!",
+		"Two times Pimpy...",
+		"Three times Bape!",
+		"Not a family tree, but a family circle.")
+
+/mob/living/simple_animal/hostile/pitbull/smashednslammed/summoned_pitbull
+	faction = "wizard" // so they get along with other wizard mobs
+	meat_type = /obj/item/weapon/ectoplasm //a magical dog
+	
```

Comment:
Unnecessary as this is already inherited from the parent.

Reply:
Ditto